www/static/plugins/supportedplatforms.jsx

```diff
@@ -6,74 +6,34 @@ var Preact = require('preact'),
 var SupportedPlatforms = createClass({
     render: function() {
         var keywords = this.props.keywords;
-        var sortedPlatforms = [
-            {present:false, icon: "android", alt:"Android"},
-            {present:false, icon: "ios", alt:"iOS"},
-            {present:false, icon: "windows", alt:"Windows"},
-            {present:false, icon: "blackberry", alt:"Blackberry"},
-            {present:false, icon: "ubuntu", alt:"Ubuntu"},
-            {present:false, icon: "firefox", alt:"FirefoxOS"},
-            {present:false, icon: "webos", alt:"webOS"},
-            {present:false, icon: "fireos", alt:"FireOS"},
-            {present:false, icon: "osx", alt:"OS X"},
-            {present:false, icon: "browser", alt:"Browser"}
-        ];
+        var sortedPlatforms = {
+            'cordova-android': {present:false, icon: "android", alt:"Android"},
+            'cordova-ios':{present:false, icon: "ios", alt:"iOS"},
+            'cordova-windows':{present:false, icon: "windows", alt:"Windows"},
+            'cordova-osx':{present:false, icon: "osx", alt:"OS X"},
+            'cordova-browser':{present:false, icon: "browser", alt:"Browser"}
+        };
```

Comment:
Objects so bit guarantee order. So to have something sorted you would have to add an extra stay of keys.

Reply:
Can you take care of this @erisu ? 

Very much needed update to the plugin page.

Reply:
@janpio Sure, I just submitted a new PR with the corrections. 

Some how I lost branch to this PR and cant update this ha. Please check out the new changes on PR #848.

@raphinesse Please also take a look if you have time.